acceptance/tests/fixtures/bases/api-gateway/kustomization.yaml

```diff
@@ -5,5 +5,5 @@ resources:
   - gatewayclassconfig.yaml
   - gatewayclass.yaml
   - apigateway.yaml
-  - httproute.yaml
+  - tcproute.yaml
```

Comment:
is there a reason we needed to change the existing setups?

Reply:
I believe goland must have gone this automatically when I was renaming things. I have reverted it.